.github/workflows/TASKLIST_CARBONISATION.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     name: Run visual regression
     runs-on: ubuntu-latest
     container:
-      image: mcr.microsoft.com/playwright:v1.34.1
+      image: mcr.microsoft.com/playwright:v1.35.1
```

Comment:
Unfortunately, this still does not fix the timeout issue (cf. https://github.com/bpmn-io/form-js/issues/674)